src/tools/cephfs/TableTool.cc

```diff
@@ -353,7 +353,7 @@ class TableHandlerOmap
     // Compose a transaction to clear and write header
     librados::ObjectWriteOperation op;
     op.omap_clear();
-    op.set_op_flags(librados::OP_FAILOK);
+    op.set_op_flags2(LIBRADOS_OP_FLAG_FAILOK);
```

Comment:
@gregfortytwo not sure, i am just following other callers of `set_op_flags2()`. for example: https://github.com/ceph/ceph/blob/master/src/test/librados/io.cc#L261